pom.xml

```diff
@@ -381,6 +387,12 @@
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
```

Comment:
mockito is part of base-test, if adding to parent project, maybe it needs to be removed from there?

Reply:
this is weird, it was not finding it.. ok this is now fixes ;)